package.json

```diff
@@ -57,6 +57,7 @@
     "js-cookie": "2.1.4",
     "kerberos": "0.0.23",
     "lodash": "4.17.4",
+    "leaflet": "^1.1.0",
```

Comment:
Remove the `^` carat here. We like to keep exact version imports to prevent bugs that might occur when one installation might have slightly different versions of dependencies than another one, even with the same codebase.